CoreGraphNode/nbproject/genfiles.properties

```diff
@@ -1,8 +1,8 @@
-build.xml.data.CRC32=ba351fce
+build.xml.data.CRC32=a437b10f
```

Comment:
This file change shouldn't be committed

Reply:
Fixed

Reply:
Hey @formalhaut69 and @aldebaran30701 if the genfiles have changed then they should be committed. Normally genfiles change when you add new dependencies between modules.